controllers/storagecluster/exporter.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/imdario/mergo"
 	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
 	ocsv1 "github.com/red-hat-storage/ocs-operator/api/v1"
+	"golang.org/x/exp/maps"
```

Comment:
From the package doc
```
Warning: Packages here are experimental and unreliable. Some may one day be promoted to the main repository or other subrepository, or they may be modified arbitrarily or even disappear altogether.
```
Might not be a good idea to use it.

Reply:
using mergo